platform/country-converter/platform-country-converter-api/src/main/java/org/codice/ddf/country/converter/api/CountryCodeConverter.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.country.converter.api;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/**
+ * Service to convert country codes into various formats. In order to convert country codes from
```

Comment:
Should this description stay in the interface or be moved to the implementation class, LocalCountryCodeConverter? The reason why it's currently in the interface, is so that when an integration wants to use the API, he'll know exactly how to use it without knowing what the implementation is, otherwise, it may be difficult for the integrator to know what the values of the sourceFormat and desiredFormat should be as they're just strings.